src/Compilers/Core/Portable/ReferenceManager/CommonReferenceManager.State.cs

```diff
@@ -380,11 +380,21 @@ internal bool IsBound
             Interlocked.Exchange(ref _isBound, 1);
         }
 
+        /// <summary>
+        /// Global namespaces of assembly references that have been superseded by an assembly reference with a higher version are 
+        /// hidden behind <see cref="SupersededAlias"/> to avoid ambiguity when they are accessed from source.
+        /// All existing aliases of a superseded assembly are discarded.
+        /// </summary>
+        private static readonly ImmutableArray<string> SupersededAlias = ImmutableArray.Create("<superseded>");
```

Comment:
Should "<superseded>" be exposed as a public constant so that consumers of the public API could use it?